test/e2e/servicebackend/service_backend.go

```diff
@@ -70,9 +71,9 @@ var _ = framework.IngressNginxDescribe("Service backend - 503", func() {
 
 		f.EnsureIngress(bi)
 
-		f.WaitForNginxServer(host,
+		f.WaitForNginxConfiguration(
```

Comment:
why are you changing these?

Reply:
because the change in the template means there is no `return 503` (we only have the name of the server

Reply:
I understand that, I meant WaitForNginxServer > WaitForNginxConfiguration change

Reply:
These assertions will pass even without your template changes.

Reply:
You could do something like

```
f.WaitForNginxServer(host,
    func(server string) bool {
        return strings.Contains(server, "proxy_pass http://upstream_balancer;")
    })
```

Reply:
done